partiql-lang/src/main/kotlin/org/partiql/lang/eval/Bindings.kt

```diff
@@ -56,6 +56,63 @@ fun PartiqlAst.CaseSensitivity.toBindingCase(): BindingCase = when (this) {
     is PartiqlAst.CaseSensitivity.CaseSensitive -> BindingCase.SENSITIVE
 }
 
+/**
+ * Represents a namespaced global binding.
+ */
+class BindingId(
```

Comment:
This class isn't used by anything else in this file--consider moving it to its own file.